tests/common.py

```
@@ -597,12 +597,13 @@ def _assertNoHolidays(self, instance_name, *args):
         self.assertEqual(0, len(holidays))
 
     # LocalizedHolidays.
-    def _assertLocalizedHolidays(self, localized_holidays, language=None):
+    def _assertLocalizedHolidays(self, localized_holidays, language=None, categories=None):
```

Ruff (0.15.9)

[warning] Missing return type annotation for private function `_assertLocalizedHolidays`

Add return type annotation: `None`

(ANN202)